tools/updIndex.rb

```diff
@@ -8,11 +8,14 @@
 
 # Remainder are the requirements for this program
 require 'aws-sdk'
-require 'pp'
 
 # Run from the right directory (the parent of the tools dir)
 Dir.chdir(File.dirname(File.expand_path(File.dirname(__FILE__))))
 
+# I had to run script first time to define this as an analysis scheme, 
+#  then ran script again to then call it as an author option
+analysis_schemes = [ { analysis_scheme_name: "no_stemming", analysis_options: { algorithmic_stemming: "none" }, analysis_scheme_language: "en" } ]
```

Comment:
So what process should we use to do this on stg and prd? Comment out the author option, run it, then comment back in?

Reply:
I'll write the code up better so the whole thing does this in two phases